test

```diff
@@ -74,13 +74,18 @@ if [ -n "${vetRes}" ]; then
 	exit 255
 fi
 
-if command -v go-nyet >/dev/null 2>&1; then
-  echo "Checking go-nyet..."
-  nyetRes=$(go-nyet -exitWith 0 $FMT)
-  if [ -n "${nyetRes}" ]; then
-          echo -e "go-nyet checking failed:\n${nyetRes}"
-          exit 255
-  fi
-fi
+echo "Checking govet -shadow..."
+for path in $FMT; do
```

Comment:
It is helpful to check the format of go files in main directory IMO.

`go tool vet` doesn't support to check directories and files in one command line. If they are put in different command lines, that is fine.